Old style exceptions --> new style

cclauss opened the pull request:
Old style exceptions were dropped from Python 3 because the syntax was too ambiguous.

Commit:
Old style exceptions --> new style

Old style exceptions were dropped from Python 3 because the syntax was too ambiguous.

jnothman commented:
It would be better to just replace this file with a more recent version of pyparsing

Commit:
Fixed the raising of exceptions also

http://python-future.org/compatible_idioms.html#raising-exceptions

#10513 is still the more complete approach.

TomDLT commented:
cf #10513
